components/ProfileInformation/ProfileInformation.tsx

```diff
@@ -103,7 +103,7 @@ export const ProfileInformation = ({
     }
 
     useEffect(() => {
-        if (isFocused) {
+        if (isFocused && profile == undefined) {
             getProfileInformation()
```

Comment:
Detta gör att man inte laddar om profilinformationen om man kommer till sidan genom att gå bakåt. Så man laddar bar in ny info om profile är undefined.